gateway-api/update.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -eoux pipefail
+
+VERSION="v1.1.0"
```

Comment:
I planned to use version 1.0 for this PR, but I obviously forgot to change it here. (: